pages/tianhaopx.md

```diff
@@ -0,0 +1,22 @@
+## Tianhao Guo
+
+#### Worcester, MA, EDT
+
+#### macOS Sierra 10.12
+
+I am a master student major in Computer Science at Worcester Polytechnic Institute.
+
+Here's my skills:
+
+- Java
+- Python
+- JavaScript
+- MySQL
+- MongoDB
+- Hadoop
+- Linux/Unix
+- Git
+
+I'm a team player, and I'm excited to joining you guys!
+
+Connect me on Linkedin: https://www.linkedin.com/in/tianhaoguo
```

Comment:
Could you make `Connect with me on Linkedin` send you to the link instead of pasting it there?